python/ray/tune/result.py

```diff
@@ -29,8 +29,8 @@
     # (Required) Accumulated timesteps for this entire experiment.
     "timesteps_total",
 
-    # (Optional) If training is finished.
-    "done",
+    # (Optional) If training is terminated.
+    "terminated",
```

Comment:
We shouldn't rename this unless there's a good reason since it's part of the public API